### PR TITLE
GitHub Action to check if mocks are synced with wandb/client:master

### PR DESCRIPTION
**Pull request by dmitryduev**
Tested this by screwing up `mock_server.py` -- it works (see https://github.com/wandb/yea-wandb/pull/39/commits/0c6d6840550cf14dda7027311304d8c03ecb9aa8).

### CODE REVIEW

**Comment by kptkin** [comment]
Ah this is a nice addition, I wonder if we could have something that automatically syncs them.
Or some way not to maintain 2 copies

**Comment by dmitryduev** [comment]
@kptkin ah, good thinking! like downloading/checking sync as a part of the plugin setup process?

**Comment by dmitryduev** [comment]
Idea: How about a GH action in `wandb/client` that opens a PR here if those files are touched on master? @raubitsj?

**Comment by dmitryduev** [comment]
This: https://github.com/marketplace/actions/create-pull-request

File: .github/workflows/are-mocks-synced.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        run: |
+          wget https://raw.githubusercontent.com/wandb/client/master/tests/utils/mock_requests.py
+          diff mock_requests.py src/yea_wandb/mock_requests.py
+        continue-on-error: true
+      - name: Check if mock_server is up to date with wandb/client:master
+        id: mock_server
+        run: |
+          wget https://raw.githubusercontent.com/wandb/client/master/tests/utils/mock_server.py
+          diff mock_server.py src/yea_wandb/mock_server.py
+        continue-on-error: true
```
Comment (raubitsj) [inline .github/workflows/are-mocks-synced.yml:26]: need artifacts_emu also.

**Comment by dmitryduev** [comment]
Closing in favor of https://github.com/wandb/client/pull/3189 -- it seems to make more sense to have this check there.

**Pull request closed without merging**